tests/test_admin.py

```diff
@@ -1030,6 +1031,33 @@ def test_menuitem_delete_view_root(self):
         self.assertEqual(response.status_code, 200)
         self.assertEqual(MenuItem._base_manager.count(), 1)
 
+    def test_menuitem_delete_view_breadcrumbs(self):
+        """
+        Breadcrumbs for the delete view should render valid url's
+        """
+        menu_content = factories.MenuContentWithVersionFactory(version__created_by=self.user)
+        child = factories.ChildMenuItemFactory(parent=menu_content.root)
+
+        # Get the url for deleting a single URL
+        delete_url_single = reverse(
+            "admin:djangocms_navigation_menuitem_delete", args=(menu_content.id, child.id,)
+        )
+
+        with self.login_user_context(self.user):
+            # Hit the confirmation page using get request
+            confirmation_response = self.client.get(
+                delete_url_single, data={"menu_content_id": menu_content.id}
+            )
+
+        # Get all the hrefs in the markup for the breadcrumbs div
+        soup = BeautifulSoup(confirmation_response.content, features="lxml")
+        breadcrumb_html = soup.find("div", class_="breadcrumbs")
+        for a in breadcrumb_html.find_all('a', href=True):
+            with self.login_user_context(self.user):
+                breadcrumb_url = a['href']
+                response = self.client.get(breadcrumb_url)
+                self.assertEqual(response.status_code, 200)
```

Comment:
Normally we would check that the url is what we expect, I guess not throwing a 404 is ok, although it could give a 200 and be the wrong url. Fine to leave as is.

Reply:
@Aiky30 Updated the test to validate the content of each url. Also looks like we need lxml for the CI to pass.